src/controller/StatusController.hpp

```diff
@@ -1,71 +0,0 @@
-/*
```

Comment:
Let's keep the status controller that provides health and readiness routes for k8s